curl/libcurl-build.sh

```diff
@@ -31,14 +31,16 @@ alert="\033[0m${red}\033[1m"
 alertdim="\033[0m${red}\033[2m"
 
 # Set trap to help debug any build errors
-trap 'echo -e "${alert}** ERROR with Build - Check /tmp/curl*.log${alertdim}"; tail -30 /tmp/curl*.log' INT TERM EXIT
+trap 'echo -e "${alert}** ERROR with Build on line $LINENO ($0) - Check /tmp/curl*.log${alertdim}"; tail -30 /tmp/curl*.log' ERR TERM EXIT
+trap 'exit 1' INT
```

Comment:
Updated traps to catch user cancellation and to provide more details during build errors (line number and file).